mender-convert

```diff
@@ -103,9 +103,10 @@ Examples for expert actions:
 
         ./mender-convert raw-disk-image-create-partitions
                 --raw-disk-image <raw_disk_image_path>
-                --mender-disk-image <mender_image_name>
+                [--mender-disk-image <mender_image_name>]
+                --artifact-name release-1_1.5.0
                 --device-type <beaglebone | raspberrypi3>
-                --data-part-size-mb 128
+                [--data-part-size-mb 128]
```

Comment:
This change seems to be unrelated. Intentional?

Reply:
Yes, intentional. I marked --data-part-size-mb as optional just like --mender-disk-image.